recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
   run:
     - python
     - alembic >=1.4
+    - importlib-metadata >=3.6
```

Comment:
This should be for # [py < 310] (see https://github.com/jupyterhub/jupyterhub/blob/3.1.0/requirements.txt#L4)

Reply:
Thanks, Charles. Could you clarify what changes are needed for this in particular?

Reply:
Sure, this line:
`importlib_metadata>=3.6; python_version < '3.10'`
Translates to this in a conda recipe:
`- importlib_metadata >=3.6. # [py<310]`

Reply:
@cbouss @varlackc It seems that changes to the pinning for this dependency causes test failures and therefore build failures for all archetypes. See [here.](https://github.com/AnacondaRecipes/jupyterhub-feedstock/pull/15/commits/2314bb248f0f112d7b3c8dbde151dc4ec4c4dfbc) 
This change returns a `Module not found` error for `importlib_metadata`. Would reconsidering or modifying this pinning be an option?